packages/barclay-integration/src/BarclaycardPaymentMethod.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { FunctionComponent } from 'react';
+
+import { HostedPaymentComponent } from '@bigcommerce/checkout/hosted-payment-integration';
+import {
+    PaymentMethodProps,
+    PaymentMethodResolveId,
+    toResolvableComponent,
+} from '@bigcommerce/checkout/payment-integration-api';
+
+const BarclaycardPaymentMethod: FunctionComponent<PaymentMethodProps> = ({
+    checkoutService,
+    checkoutState,
+    method,
+    paymentForm,
+    ...rest
+}) => {
+    return (
+        <HostedPaymentComponent
```

Comment:
I believe that in we need to create common component for such integrations. We already have couple providers that could reuse same component. We need to create follow up ticket for that